packages/nimble-components/src/theme-provider/tests/theme-provider.stories.ts

```diff
@@ -0,0 +1,50 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import { hiddenWrapper } from '../../utilities/tests/hidden';
+
+const metadata: Meta = {
+    title: 'Internal/Theme Provider',
+    parameters: {
+        docs: {
+            description: {
+                component: ''
+            }
+        }
+    }
+};
+
+export default metadata;
+
+const langDescription = `Defines the language of the element. See [external documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) for details.
+
+Users should set \`lang\` on the root \`html\` element of the page to reflect the language of the content. If necessary, users may override the language for a subtree by inserting a \`nimble-theme-provider\` element and setting its \`lang\` attribute. Nimble elements will not honor a \`lang\` value set on any other type of ancestor element.`;
+
+export const themeProvider: StoryObj = {
+    render: createUserSelectedThemeStory(hiddenWrapper(html`<style></style>`)),
```

Comment:
This is probably feature creep, so feel free to reject. But it'd be cool to populate the HTML with a theme provider and some elements that respect the tokens set on it (like a table with a date column for the `lang` attribute and maybe also some content that uses background tokens that respect the `theme` attribute). Then the Storybook controls could affect the theme provider and users could see the impact interactively.

Reply:
I just added a table, as that reflects the `theme` as well as the `lang`. I don't know how to demo the `direction` setting, but I left it selectable anyway.